migrator/zenodo_rdm_migrator/stream.py

```diff
@@ -117,8 +118,8 @@
 
 ActionStreamDefinition = StreamDefinition(
     name="action",
-    extract_cls=JSONLExtract,
-    transform_cls=ZenodoTransactionGroupTransform,
-    load_cls=ZenodoTransactionLoad,
+    extract_cls=KafkaExtract,
+    transform_cls=Tx,
```

Comment:
minor: sorry, forgot to update this, it should point to a class extending `BaseTxTransform`, with whatever transform actions we have implanted configured in the `actions` class variable.

Reply:
Fixed, I did not export `ZenodoTxTransform` in the root module `transform` because it causes a circular import which I'm not a 100% sure how to solve (none of the options seem good to me). It's caused by the fact that the `actions/drafts.py` imports the root `ZenodoRecordTransform`.